Applications/Utils/GeoTools/MoveGeometry.cpp

```diff
@@ -0,0 +1,85 @@
+/**
+ * \file MoveGeometry.cpp
+ * \author Karsten Rink
+ * \date   2015-08-04
+ * \brief  A small tool to translate geometries.
+ *
+ * \copyright
+ * Copyright (c) 2012-2015, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+// ThirdParty
+#include "tclap/CmdLine.h"
+
+// ThirdParty/logog
+#include "logog/include/logog.hpp"
+
+// BaseLib
+#include "LogogSimpleFormatter.h"
+
+// MathLib
+#include "MathLib/Vector3.h"
+
+// FileIO
+#include "FileIO/XmlIO/Qt/XmlGmlInterface.h"
+
+// GeoLib
+#include "GeoLib/GEOObjects.h"
+
+#include <QtGui/QApplication>
+
+int main(int argc, char *argv[])
+{
+	QApplication app(argc, argv);
```

Comment:
I don't know too much about qt. Can you explain, why this app, which is not used later, is needed?